x-pack/plugins/ingest_manager/server/services/epm/packages/install.ts

```diff
@@ -230,12 +233,11 @@ export async function createInstallation(options: {
 export const saveInstalledKibanaRefs = async (
   savedObjectsClient: SavedObjectsClientContract,
   pkgName: string,
-  installedAssets: AssetReference[]
+  installedAssets: KibanaAssetReference[]
 ) => {
-  await savedObjectsClient.update(PACKAGES_SAVED_OBJECT_TYPE, pkgName, {
+  return savedObjectsClient.update(PACKAGES_SAVED_OBJECT_TYPE, pkgName, {
```

Comment:
Is this the same return type as before? Does update return installedAssets 

Is the removal of await intentional/required. They're almost the same thing, but if it's not a requirement, I rather leave things as-is

Reply:
yes its intentional as I am no longer returning the installed assets here because I need to return it from `installKibanaAssets` instead.  It could still return it, but it was not actually being used to begin with.

Reply:
updated because had to make another change anyway